src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -305,6 +306,27 @@ public boolean apply(StepDescriptor i) {
         return r;
     }
 
+    /** Return true if we can easily create a nice String for user display from the object */
+    static boolean isAbleToUseToStringForDisplay(@CheckForNull Object o) {
```

Comment:
Prefer that arguments be `@Nonnull` unless there is a compelling reason why it would be cumbersome for callers to do their own null handling, or there is some meaningful need to pass nulls (which apparently here there is not).